github/commitstatus.go

```diff
@@ -175,9 +181,29 @@ func httpRequestDo(req *http.Request) (httpResponse, error) {
 	response.oauthInfo = fmt.Sprintf("X-Accepted-Oauth-Scopes: %v, X-Oauth-Scopes: %v",
 		resp.Header.Get("X-Accepted-Oauth-Scopes"), resp.Header.Get("X-Oauth-Scopes"))
 
-	response.rateLimitRemaining, _ = strconv.Atoi(resp.Header.Get("x-ratelimit-remaining"))
-	rateLimitReset, _ := strconv.Atoi(resp.Header.Get("x-ratelimit-reset"))
-	response.rateLimitReset = time.Unix(int64(rateLimitReset), 0)
+	if limit := resp.Header.Get("X-RateLimit-Remaining"); limit != "" {
```

Comment:
I took some inspiration from: https://github.com/google/go-github/blob/9e4ce880cd4db6c8f8459806a8c1276a14db3983/github/github.go#L665

It seems that its perfectly valid and possible that these two headers (`X-RateLimit-Remaining`, `X-RateLimit-Reset`) are not reported. But if they are reported and we fail to convert their value to integer we must report an error.
In the `go-github` pkg they are happy enough with return value of 0 and ignore the error, I prefer to be explicit about it.